packages/remix-react/browser.tsx

```diff
@@ -55,7 +71,12 @@ export function RemixBrowser(_props: RemixBrowserProps): ReactElement {
         future: window.__remixContext.future,
       }}
     >
-      <RouterProvider router={router} fallbackElement={null} />
+      <RemixErrorBoundary
+        location={location}
+        component={RemixRootDefaultErrorBoundary}
+      >
+        <RouterProvider router={router} fallbackElement={null} />
+      </RemixErrorBoundary>
```

Comment:
Wrap everything in a last-resort error boundary, and send the router location in so users can back-button out of it